components/profile_traits/time.rs

```diff
@@ -160,3 +167,9 @@ pub fn send_profile_data(
 ) {
     profiler_chan.send(ProfilerMsg::Time((category, meta), (start_time, end_time)));
 }
+
+pub(crate) fn elapsed_since_epoch() -> Duration {
+    SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap_or_default()
+}
```

Comment:
I would just call `SystemTime::now()` instead of exporting this new function. Perhaps it makes sense to have a series of helpers that make dealing with time easier, but I'm not sure the `profile_traits` crate is the right place for this.

Reply:
Thanks for the feedback! I have gotten rid of the offending function. 

When you say "just call `SystemTime::now()`", do you mean the whole `SystemTime::now().duration_since(UNIX_EPOCH)...`? Because I can't see a way to get nanoseconds from the SystemTime directly, even through the UNIX_EPOCH is just a zero value for this particular struct. Am I missing something?